SomerenUI/EditStudentForm.cs

```diff
@@ -72,38 +61,12 @@ private void btnUpdateStudent_Click(object sender, EventArgs e)
             }
         }
 
-        private void ValidateInputs(out string firstName, out string lastName, out string phoneNumber)
+        private int ValidateStudentNumber(string studentNumberString)
         {
-            if (txtFirstName.Text.IsNullOrEmpty())
-                throw new Exception(Properties.Resources.ErrorMessageNoFirstName);
-            else
-                firstName = txtFirstName.Text;
-
-            if (txtLastName.Text.IsNullOrEmpty())
-                throw new Exception(Properties.Resources.ErrorMessageNoLastName);
-            else
-                lastName = txtLastName.Text;
-
-            phoneNumber = txtPhoneNumber.Text;
-        }
-
-        private void ValidateInputs(out int studentNumber, out string firstName, out string lastName, out string phoneNumber, out string classNumber)
-        {
-            if (txtFirstName.Text.IsNullOrEmpty())
-                throw new Exception(Properties.Resources.ErrorMessageNoFirstName);
-            else
-                firstName = txtFirstName.Text;
-
-            if (txtLastName.Text.IsNullOrEmpty())
-                throw new Exception(Properties.Resources.ErrorMessageNoLastName);
-            else
-                lastName = txtLastName.Text;
-
-            phoneNumber = txtPhoneNumber.Text;
-
-            if (txtStudentNumber.Text.Length > 6)
+            int studentNumber;
+            if (studentNumberString.Length != 6)
                 throw new Exception(Properties.Resources.ErrorMessageStudentNumberLength);
-            else if (!int.TryParse((txtStudentNumber.Text), out int tryGetStudentNumber))
+            else if (!int.TryParse((studentNumberString), out int tryGetStudentNumber))
```

Comment:
I would rename the out variable to studentNumber or so. It's variable and not an action/method.

Reply:
Also, use const for 6 at line 67, please.